src/js/modules/components/moveable.js

```diff
@@ -1,15 +1,26 @@
 export const MOVE_END = 'moveend';
 const PREVENT_SELECT_CLASS = 'noselect';
 
-export function moveableComponent({ view }) {
+export function moveableComponent({ view, staticMode = false }) {
```

Comment:
I'm not exactly a fan of passing `staticMode` here - it will be used only in `VIEWER` so we should just check this when during creation of `moveableComponent` and not pass it here. There is no method to change this behaviuor during lifetime of `moveableComponent`.